.github/workflows/mr-veileder-flate.yaml

```diff
@@ -64,24 +62,69 @@ jobs:
       - name: Set up gcloud
         uses: google-github-actions/setup-gcloud@v2
       - name: Upload files to GCS
-        run: gsutil -m rsync -r /home/runner/work/mulighetsrommet/mulighetsrommet/frontend/mulighetsrommet-veileder-flate/dist gs://mulighetsrommet-veilederflate-dev
+        run: gsutil -m rsync -r frontend/mulighetsrommet-veileder-flate/dist gs://mulighetsrommet-veileder-flate-demo
 
-  deploy-demo:
-    name: Deploy (demo)
+  deploy-modia-dev:
+    name: Deploy (modia-dev)
     runs-on: ubuntu-latest
     needs: [ci]
     permissions:
-      id-token: "write" # Used to authenticate with Google Cloud
+      id-token: "write" # Used to authenticate with Google Cloud (through cdn-upload)
     if: github.event_name == 'push' && github.ref_name == 'main' || github.event_name == 'workflow_dispatch'
     steps:
       - uses: actions/checkout@v4
       - name: Setup frontend
         uses: ./.github/actions/setup-frontend
       - name: Build
-        run: npx turbo run build:demo --filter=mulighetsrommet-veileder-flate
+        run: npx turbo run build:modia --filter=mulighetsrommet-veileder-flate -- --base=https://cdn.dev.nav.no/mulighetsrommet/arbeidsmarkedstiltak-modia/dist
+        env:
+          VITE_FARO_URL: https://telemetry.ekstern.dev.nav.no/collect
+      - name: Upload to NAV CDN
+        uses: navikt/frontend/actions/cdn-upload/v1@main
```

Comment:
Laster nå  opp modia-appen til frontend-platttform CDN i stedet. Denne kan refereres til fra veilarbpersonflate og vi kan dermed bli kvitt instanser av poao-frontend for modia-varianten av applikasjonen (blir gjort i en senere PR).